robotiq_force_torque_sensor/src/rq_sensor_state.cpp

```diff
@@ -154,14 +154,27 @@ static void rq_state_start_stream()
  */
 static void rq_state_run()
 {
+	static unsigned char retries = 0;
```

Comment:
If retries always gets set back to zero in the code below, then why declare it as static.  It would be better to declare it as a non-static variable with an initial value of zero.

In addition, I would use a larger size integer.  It would be very easy to specify a number of retries that cannot be represented by the `unsigned char` type.  The compiler may or may not detect such an issue (I'm not sure).